Makefile

```diff
@@ -28,6 +28,33 @@ all: $(TARGET)
 test: LDFLAGS += $(TEST_COV)
 test: integration_test
 
+# Build Lightning binary
+build: Dockerfile
+	# Create image for compiling Lightning under Ubuntu 14.04/Trusty
```

Comment:
Should these indents be 2 spaces, or not since we are in a Makefile and not in C++ code?